### PR TITLE
Add release workflow for helm charts

### PR DESCRIPTION
**Pull request by eb-oss**
Add a reusable workflow for pushing ONOS helm charts.

### CODE REVIEW

**Review by SeanCondon — DISMISSED** [review]
LGTM

File: workflows/release-helm-charts.yml
Comment (gab-arrobo) [inline workflows/release-helm-charts.yml:27]: It looks like this target exists in the `sdran-helm-charts` but I do not see it in the `roc-helm-charts`, correct? Or am I overlooking something?

File: workflows/release-helm-charts.yml
Comment (gab-arrobo) [inline workflows/release-helm-charts.yml:24]: Is not there already a GHA for `yq`? Also, any reason to install it using snap instead of using its source?

Comment (eb-oss) [reply workflows/release-helm-charts.yml:24]: This was based on [sdran-helm-charts' release action](https://github.com/onosproject/sdran-helm-charts/blob/master/.github/workflows/release.yml), but you're completely correct. I've removed the snap installations and replaced them with an action.

Comment (eb-oss) [reply workflows/release-helm-charts.yml:24]: Replaced with an action.

File: workflows/release-helm-charts.yml
Comment (eb-oss) [reply workflows/release-helm-charts.yml:41]: Replaced with an action.

File: workflows/release-helm-charts.yml
Comment (eb-oss) [reply workflows/release-helm-charts.yml:27]: Nope, you're correct. I've been working a lot on Voltha, which uses a standardized library of makefiles, and just totally forgot to double-check here. But it's a simple call to the version_check.sh script, so I just replaced it with a direct call.

File: workflows/version_check.sh
Comment (eb-oss) [reply workflows/version_check.sh:1]: I believe it is a more thorough check than the inline script in sdcore-helm-charts/.../push.yml, yes. We can certainly sync up the two down the line, though that's outside of the scope of this PR.

**Review by gab-arrobo — APPROVED** [review]
Overall, it looks good to me.

@eb-oss, please add branch protection to the default branch such that PRs can be merged only after passing all required checks and at least has one approval. Additionally, are you planning to enable Dependabot to this repo such that newer versions of the GHA get monitored?

**Comment by eb-oss** [comment]
> @eb-oss, please add branch protection to the default branch such that PRs can be merged only after passing all required checks and at least has one approval. Additionally, are you planning to enable Dependabot to this repo such that newer versions of the GHA get monitored?

Branch protections and dependabot alerts have been enabled.

**Pull request merged**
